torch/fx/passes/backends/nvfuser/operator_support.py

```diff
@@ -0,0 +1,180 @@
+import typing as t
+
+import torch
+import torch.fx
+from torch.fx.passes.operator_support import OperatorSupport
+from torch.fx.passes.tools_common import CALLABLE_NODE_OPS
+
+try:
+    from torch._C._nvfuser import FusionDefinition as fd  # type: ignore[import]
+except ImportError:
+    # Not all PyTorch builds have nvfuser
+    pass
+
+class NvFuserOperatorSupport(OperatorSupport):
```

Comment:
This class merges two unrelated things: nvFuser support of TorchScript and nvFuser support of primTorch. TorchScript graph is partitioned by nvFuser and we don't need to think about TorchScript here.

Reply:
removed prim ops to avoid confusion.